src/cases/utils/utils.ts

```diff
@@ -60,7 +60,8 @@ export class Utils {
   public static hasMatchedJurisdictionAndCaseType(featureVariation: FeatureVariation, jurisdictionId: string, caseType: string): boolean {
     if (featureVariation.jurisdiction === jurisdictionId) {
       if ((featureVariation?.caseType === caseType) ||
-        (featureVariation.includeCaseTypes?.length > 0 && featureVariation.includeCaseTypes.includes(caseType))) {
+        (featureVariation?.includeCaseTypes?.length > 0 &&
+          featureVariation?.includeCaseTypes.some((ct) => ct === caseType || new RegExp('^(sscs-pr-|SSCS-PR-)\\d+$').test(ct)))) {
```

Comment:
That wasn't what I meant. The regex should be in the config, not the code

Reply:
regEx added in LD